include/aspect/gravity_model/vertical.h

```diff
@@ -37,7 +37,7 @@ namespace aspect
      * @ingroup GravityModels
      */
     template <int dim>
-    class Vertical : public Interface<dim>
+    class Vertical : public Interface<dim>, public virtual SimulatorAccess<dim>
```

Comment:
why does the derivation have to be `virtual` here?

Reply:
You are right, I copied that and forgot to remove it.